spec/predictor_spec.rb

```diff
@@ -3,9 +3,9 @@
 require 'spec_helper'
 
 describe Crystalball::Predictor do
-  subject(:predictor) { described_class.new(instance_double('Crystalball::ExecutionMap', cases: cases), repository) }
+  subject(:predictor) { described_class.new(instance_double('Crystalball::ExecutionMap', cases: cases), repository, from: 'HEAD^', to: 'HEAD') }
```

Comment:
why do we need this?

Reply:
You are right. we don't need this.
About https://github.com/toptal/crystalball/pull/84#discussion_r194056983
comparing objects doesn't work because we initialize these objects with different `objectish`. We need to compare `sha` here.